linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -111,6 +121,29 @@ public static void main(String[] arguments)
     }
   }
 
+  private static CommandLine parseCommandLine(String[] arguments) {
+    Options options = new Options();
+    options.addOption("send-analytics", false, "track results from each run");
```

Comment:
Would you make this more explicit about this tool uploading the result to Google Analytics.

Reply:
what language do you suggest? This needs to be a short string for help, not complete documentation.